src/Site/views/languageforge/container/languageforge.html.twig

```diff
@@ -81,7 +81,7 @@
                     <breadcrumbs id="top" class="breadcrumbs d-none d-md-block"></breadcrumbs>
                 </div>
                 <ul class="nav navbar-nav">
-                    <li class="nav-item" ng-if="$ctrl.project" id="settings-dropdown-button" uib-dropdown data-ng-class="{'app-settings-available': $ctrl.header.settings.length > 0 && $ctrl.currentUserIsProjectManager}">
+                    <li class="nav-item" ng-if="$ctrl.isLexiconProject" id="settings-dropdown-button" uib-dropdown data-ng-class="{'app-settings-available': $ctrl.header.settings.length > 0 && $ctrl.currentUserIsProjectManager}">
```

Comment:
I see how you tried to combine the "show/hide" into a single boolean `isLexiconProject` but this won't work here since you have conflated the logic of the share button into the isLexiconProject boolean.

I think what we really need is a displayShareButton and displaySettingsButton whose logic contains the right combination of switches.  I know that the logic is a bit fuzzy and not entirely clear from the PR.

Reply:
I have fixed this up to my own satisfaction, so I will merge when tests are passing.